cmds/gpt/gpt.go

```diff
@@ -56,20 +56,26 @@ func main() {
 		log.Fatal(err)
 	}
 
+	blockSize, err := gpt.GetBlockSize(n)
```

Comment:
what does this do if the "device" is a file, as in a disk image?

Reply:
Hm I think `ioctl` fails in that case - is that how we would usually use the gpt command?